compiler/src/dotty/tools/dotc/transform/LazyVals.scala

```diff
@@ -445,26 +721,37 @@ class LazyVals extends MiniPhase with IdentityDenotTransformer {
 object LazyVals {
   val name: String = "lazyVals"
   val description: String = "expand lazy vals"
-
   object lazyNme {
     import Names.TermName
     object RLazyVals {
       import scala.runtime.LazyVals.{Names => N}
+      val waiting: TermName                = N.waiting.toTermName
```

Comment:
As mentioned in the runtime lib, these should be compiler-only knowledge, so
```suggestion
      val waiting: TermName                = "waiting".toTermName
```